helpers/containsUser.ts

```diff
@@ -0,0 +1,20 @@
+import { UserDto } from "../dto/userDto";
+import { UserDtoResponse } from "../dto/userDtoResponse ";
+
+export function containsUser(
+  obj: UserDto | UserDtoResponse,
+  list: UserDtoResponse[] | UserDto[],
```

Comment:
rewrite it using more consistent view: 
obj: UserDto | UserDtoResponse,
  list: UserDto[] | UserDtoResponse[],
  
  or
  
  obj: UserDtoResponse | UserDto,
  list: UserDtoResponse[] | UserDto[],